src/eventListeners/onImageRenderedBrushEventHandler.js

```diff
@@ -5,7 +5,7 @@ import external from '../externalModules.js';
 import BaseBrushTool from '../base/BaseBrushTool.js';
 import { getNewContext } from '../util/drawing.js';
 
-const brushState = store.modules.brush;
+const { state, getters, setters } = store.modules.brush;
```

Comment:
Like the use of destructors here. I think going forwards we should follow this pattern in any file that only refers to one store module 👍 .